src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -58,7 +58,7 @@
     @end
 @end
 
-@private methodSampleCode(apiMethod)
+@snippet methodSampleCode(apiMethod)
```

Comment:
In this file, the `@sampleCode` function that selects which samples to output should key off the calling form rather than what it currently does, so that both the in-code and standalone samples key off the calling form.

[Incidentally, that's part of the reason in the past I've gravitated to making these changes in 2 PRs per language: one for the generator infrastructure in Java, and a barebones stub template, and another one normalizing the standalone and in-code samples to both key off the calling form, with the in-code samples yielding the same code as before.]

Reply:
Ack.